packages/common/src/users/_internal/UserUiSchemaSettings.ts

```diff
@@ -16,6 +17,31 @@ export const buildUiSchema = async (
   options: any,
   context: IArcGISContext
 ): Promise<IUiSchema> => {
+  let communityOrgName;
+  // default Notice action
+  const orgNoticeActions = [
+    {
+      ariaLabel: `{{${i18nScope}.notice.actions.goToEOrg:translate}}`,
+      label: `{{${i18nScope}.notice.actions.goToEOrg:translate}}`,
+      icon: "launch",
+      href: `${context.portalUrl}/home/organization.html?tab=general#settings`,
+      target: "_blank",
+    },
+  ];
+  if (context.communityOrgId) {
+    // set the community org name
+    communityOrgName = context.trustedOrgs.find(
+      (org) => org.to.orgId === context.communityOrgId
+    ).to.name;
+    // add the community org action
+    orgNoticeActions.push({
+      ariaLabel: `{{${i18nScope}.notice.actions.goToCommunityOrg:translate}}`,
+      label: `{{${i18nScope}.notice.actions.goToCommunityOrg:translate}}`,
+      icon: "launch",
+      href: `${context.communityOrgUrl}/home/organization.html?tab=general#settings`,
```

Comment:
I might be completely mis-remembering our conversation, but wasn't this causing issues to link out to settings directly?

Reply:
It does, I haven't discussed the issue with Ike yet :)